app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/NavigationViewActivity.java

```diff
@@ -272,9 +270,7 @@ private void launchNavigationWithRoute() {
   }
 
   private boolean validRouteResponse(Response<DirectionsResponse> response) {
-    return response.body() != null
-      && response.body().routes() != null
-      && response.body().routes().size() > 0;
+    return response.body() != null && response.body().routes().size() > 0;
```

Comment:
`&& response.body().routes().size() > 0;` to `&& !response.body().routes().isEmpty();`